gcc/rust/ast/rust-fmt.h

```diff
@@ -258,10 +258,16 @@ struct FormatArgsHandle
   RustString rust_string;
 };
 
+typedef enum
+{
+  Format,
+  InlineAsm,
+} ParseMode;
```

Comment:
```suggestion
enum ParseMode
{
  Format,
  InlineAsm,
};
```

there's no need to typedef it in C++ for an enum

Reply:
and then I would change the enum to this, per my other comment on the FFI enum thing
```suggestion
enum ParseMode
{
  Format = 0,
  InlineAsm,
};
```